### PR TITLE
Add Humans.ai chain and HEART token

### PR DESCRIPTION
**Pull request by 0x963D**
This PR is to add the Humans.ai chain and HEART, the native staking and governance token of Humans.ai.

### CODE REVIEW

File: humans/chain.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "humans",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://humans.ai/",
+  "pretty_name": "humans",
+  "chain_id": "humans_1089-1",
+  "bech32_prefix": "humans",
```
Comment (JeremyParish69) [inline humans/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: humans/chain.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        "denom": "aheart"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/humansdotai/",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": ["v1.0.0"],
+    "cosmos_sdk_version": "0.46.11",
+    "consensus": {
```
Comment (JeremyParish69) [inline humans/chain.json:38]: can we copy the consensus data into the v1.0.0 version object below in the array, please?

Comment (0x963D) [reply humans/chain.json:38]: done, thank you!

File: humans/chain.json (hunk 1 above)
Comment (0x963D) [reply humans/chain.json:9]: Submitted PR: https://github.com/satoshilabs/slips/pull/1597

Comment (0x963D) [reply humans/chain.json:9]: PR merged

**Pull request merged**